stacks/camunda-ci-weblogic-12r2-oracle-18c-autosetup-7.10.12/docker-stack.yml

```
@@ -0,0 +1,16 @@
+---
+services:
+  main:
+    command: /home/camunda/start_container.sh
+    deploy:
+      restart_policy:
+        condition: on-failure
+    environment:
+      - DATABASE_HOST=${DATABASE_HOST}
+      - DATABASE_PORT=${DATABASE_PORT}
+      - DATABASE_NAME=${DATABASE_NAME}
+    image: registry.camunda.com/camunda-ci-weblogic:12R2-oracle-18c-autosetup-7.12.0-alpha6
```

please update the version

```suggestion
    image: registry.camunda.com/camunda-ci-weblogic:12R2-oracle-18c-autosetup-7.10.12
```